fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/traci/reader/StoppingPlaceReader.java

```diff
@@ -46,7 +46,7 @@ protected List<PublicTransportData.StoppingPlace> readFromStream(DataInputStream
             stoppingPlaceBuilder.stopFlags(VehicleStopMode.fromSumoInt(readIntWithType(in)));
             stoppingPlaceBuilder.stopDuration(readDoubleWithType(in));
             stoppingPlaceBuilder.stoppedUntil(readDoubleWithType(in));
-
+            // NOTE: this doesn't contain the start position of the stopping place
```

Comment:
for me, after reading this comment, it was not 100% clear if we just don't read the start position, or if it is not available via TraCI. Better write:

```
// startPos of the stop is currently not available in theTraCI subscription result (see https://sumo.dlr.de/docs/TraCI/Vehicle_Value_Retrieval.html, variable 0x73)
```

Reply:
Yes I agree, adapted the change!